packages/open-next/src/cache/tag/dynamodb-lite.ts

```diff
@@ -70,7 +70,7 @@ const tagCache: TagCache = {
 
       const tags = Items?.map((item: any) => item.tag.S ?? "") ?? [];
       debug("tags for path", path, tags);
-      return tags;
+      return tags.map((tag: string) => tag.replace(`${NEXT_BUILD_ID}/`, ""));
```

Comment:
Add the same comment in `dynamodb.ts` too?
`// We need to remove the buildId from the path`